views/demo.pug

```diff
@@ -0,0 +1,16 @@
+extends layout.pug
+
+block content
+	.container-fluid
```

Comment:
yeah, i still have to clean up the sidebar code and all the classes and all. On it.